scripts/env_setup.py

```diff
@@ -35,16 +32,13 @@ def pip_command(command, error_ok=False):
     )
     args = parser.parse_args()
 
-    packages = [os.path.dirname(p) for p in glob.glob("azure*/setup.py")]
-
     # Make sure pip is on the latest version
     pip_command("install --upgrade pip")
 
-    # Install packages
-    for package_name in packages:
-        # Use an eager upgrade strategy to make sure we have all the latest dependencies.
-        # This way we will be running into any dependency-related bugs before customers do.
-        pip_command("install -U --upgrade-strategy eager -e {}".format(package_name))
+    # Install package
+    # Use an eager upgrade strategy to make sure we have all the latest dependencies.
+    # This way we will be running into any dependency-related bugs before customers do.
+    pip_command("install -U --upgrade-strategy eager -e .")
```

Comment:
>pip_command("install -U --upgrade-strategy eager -e .") [](http://example.com/codeflow?start=4&length=55)

This has the side-effect of polluting developer pip namespaces.  After this, I can do things like `from test.common import test_alarm`

Reply:
Good catch, I'll see about fixing this

Reply:
Apparently this was ALWAYS true, this isn't a new issue. Very strange. Perhaps related to the `setup.py`.

Reply:
Of course, it would have always been true.  Going to a src folder should fix this if I'm understanding correctly.

Reply:
Yes, moving to the src model did fix this